index.js

```diff
@@ -157,14 +158,19 @@ module.exports = async function(options) {
   app.use(sitesMiddleware);
 
   const listen = require('util').promisify(app.listen.bind(app));
+  if (options.port != null) {
```

Comment:
check process.env.PORT first, that should win. It's OK to support options.port too.

Reply:
I will do that.

Is this what you want:

if (process.env.PORT){
     //set process.env.PORT
} else if (options.port != null) {
    //set options.port
} else {
   //get port from server string.
}